docs/tutorial/kymotracking.rst

```diff
@@ -0,0 +1,157 @@
+rate. This results in the following model for the connection score.
+
+.. math::
+
+    S(x, t) = N\left(x + v t, \sigma_{base} + \sigma_{diffusion} \sqrt{t}\right).
+
+Here `N` refers to a normal distribution. In addition to the model, we also have to set a cutoff, after which we deem
+peaks to be so unlikely to be connected that they shouldn't be. By default, this cutoff is set at two sigma. Scores
+outside this cutoff are set to zero which means they will not be accepted as a new point.
+
+
+Using the greedy algorithm
+--------------------------
+
+So, time to use this algorithm. First, we need to get an image to perform the tracing on. Let's grab a kymograph from
+a Bluelake file::
+
+    file = lk.File('kymograph.h5')
+    _, kymo = file.kymos.popitem()
+
+We can extract the image data with one of the `_image` commands::
+
+    data = kymo.green_image
+
+Let's have a look at what it looks like::
+
+    plt.imshow(data)
+
+It seems that our traces in this image are about 5 pixels wide, so let's set this as our `line_width`. The peaks have an
+intensity of about 5 photon counts, whereas the background fluctuates around 2-3. Let's set our `pixel_threshold` to 4.
+We also see that sometimes, a particle momentarily disappears. To still connect these in a single trace, we want to
+allow for some gaps in the connection step. Let's use a `window` size of 4 in this test. Running the algorithm is easy::
+
+    traces = track_particles_greedy(data, line_width=5, pixel_threshold=4, window=4)
+
+The result is a set of kymograph traces. We can get the determined time and coordinates of these traces from the
+`time` and `coordinate` attributes. Let's plot the first trace::
+
+    plt.plot(traces[0].time, traces[0].coordinate)
+
+Plotting all the lines is also quite easy using a Python list comprehension::
+
+    [plt.plot(trace.time, trace.coordinate for trace in traces]
+
+Note that the determined time and coordinates of the traces and coordinates are defined in pixels and therefore need to
+be converted to actual time and position using the delta time and pixel size of the kymograph. We can get the time
+between frames directly from the kymograph::
+
+    dt = (kymo.timestamps[0, 1] - kymo.timestamps[0, 0])/1e9
+
+And the pixel size as well::
+
+    pixel_to_um = kymo.json["scan volume"]["scan axes"][0]["pixel size (nm)"] / 1000
+
+This allows conversion from pixels back to position.
+
+
+Extracting summed intensities
+-----------------------------
+
+Sometimes, it can be desirable to extract pixel intensities in a region around our kymograph trace. We can quite easily
+extract these using the method `sample_from_image`. For instance, if we want to sum the pixels in a 9 pixel area around
+our first kymograph trace, we can invoke::
+
+    trace = traces[0]
+    trace.sample_from_image(data, n_pixels=4)
+
+Here `n_pixels` is the number of pixels to sum on either side of the trace. Note that data has to be the same image
+that the trace was based on.
```

Comment:
Can we give each trace a reference to the original `data`? That way users don't need to keep track to match the correct `trace` and `data`. There shouldn't be any extra overhead since the data exists anyway, we just have an extra reference.